### DISCUSSION TITLE
myblog/mastering-the-zen-of-trackpoint/

### DISCUSSION THREAD

**Discussion by giscus** [jasoneckert/jasoneckert.github.io#8] [Announcements]
# myblog/mastering-the-zen-of-trackpoint/

My personal website and blog

https://jasoneckert.github.io/myblog/mastering-the-zen-of-trackpoint/

**Comment by TedSelker** [comment]
When designing it we ran lots of experiments: If you practice by moving from one icon to the next stoping on each for 1 minute, you will already be faster than a trackpad.  
The Trackpoint (if set up correctly) keeps you from moving the cursor faster than your eyes track ... unless you press it to switching windows. At the slow speeds it goes at predictable speeds to get rid of tremor problems.  It also has something called negative inertia (think ABS brakes) built in to make it lively but  more controllable.   I
